### PR TITLE
fix cargo home

### PR DESCRIPTION
**Pull request by philipktlin**
Fix cargo home path to use $CARGO_HOME instead of $HOME/.cargo.

### CODE REVIEW

File: builds/checkin/edgelet.yaml [hunk 1]
```diff
@@ -36,15 +36,15 @@ jobs:
...
-      - bash: 'echo "##vso[task.setvariable variable=PATH;]$HOME/.cargo/bin:$PATH"'
-        displayName: Modify path
       - bash: |
           BASE_VERSION=`cat $BUILD_SOURCESDIRECTORY/edgelet/version.txt`
           VERSION="$BASE_VERSION$BUILD_BUILDNUMBER"
           echo "##vso[task.setvariable variable=VERSION;]$VERSION"
         displayName: Set Version
       - script: edgelet/build/linux/install.sh
         displayName: Install Rust
+      - bash: 'echo "##vso[task.setvariable variable=PATH;]$CARGO_HOME/bin:$PATH"'
```
Comment (arsing) [inline builds/checkin/edgelet.yaml]: You also need the fallback to `$HOME/.cargo` where `$CARGO_HOME` isn't defined here, like you did in the scripts.

Comment (philipktlin) [reply builds/checkin/edgelet.yaml]: AFAIK, install Rust should set CARGO_HOME; therefore we don't need to have fallback value in here. is it correct?

Comment (philipktlin) [reply builds/checkin/edgelet.yaml]: Yes, I need to explicitly set it in the current session.  will do.

Comment (arsing) [reply builds/checkin/edgelet.yaml]: Installing Rust does not set `CARGO_HOME`. `CARGO_HOME` is used to modify where Rust is installed. The assumption is that VSTS is setting it as part of pre-installing Rust in their VM image.

Comment (philipktlin) [reply builds/checkin/edgelet.yaml]: Updated.  Thanks.

**Pull request merged**